cmake/f3dPlugin.cmake

```diff
@@ -166,12 +166,11 @@ The `NAME` argument is required. The arguments are as follows:
 macro(f3d_plugin_build)
   cmake_parse_arguments(F3D_PLUGIN "FREEDESKTOP;FORCE_STATIC" "NAME;DESCRIPTION;VERSION" "VTK_MODULES;ADDITIONAL_RPATHS;MIMETYPE_XML_FILES;CONFIGURATION_DIRS" ${ARGN})
 
-  if (NOT F3D_PLUGIN_VTK_MODULES)
-    # Make sure we are requiring at least one module
-    list(APPEND F3D_PLUGIN_VTK_MODULES "CommonCore")
-  endif ()
-
-  find_package(VTK 9.0 REQUIRED COMPONENTS ${F3D_PLUGIN_VTK_MODULES})
+  find_package(VTK 9.0 REQUIRED COMPONENTS
+    CommonCore
+    CommonExecutionModel
+    IOImport
```

Comment:
in https://github.com/f3d-app/f3d/pull/785/files I decided to always link against IOImport and CommonExecutionModel to avoid potential issues where plugin developpers do not list modules correctly.

I can change that back if you think this is better but since F3D require IOImport and CommonExecutionModel this is fine imo.

Reply:
Ok I'm fine with that choice